lib/features/bulk_log.js

```diff
@@ -58,9 +59,14 @@ BulkLog.prototype = {
       }
     );
 
-     var stat = yield fs.stat.bind(fs, this.file.path);
+    let stat = await new Promise((accept, reject) => {
```

Comment:
Thanks for pointing out mz, didn't look at it before...
fixed in 56188c9248b3bb0cfc6b78a75fb5b6334af0b59f